eng/Version.Details.xml

```diff
@@ -5,9 +5,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>15f00efd583eab4372b2e9ca25bd80ace5b119ad</Sha>
     </Dependency>
-    <Dependency Name="Private.SourceBuild.ReferencePackages" Version="1.0.0-beta.20278.2">
+    <Dependency Name="Private.SourceBuild.ReferencePackages" Version="1.0.0-beta.20367.1">
       <Uri>https://github.com/dotnet/source-build-reference-packages</Uri>
-      <Sha>f3db8b2a4a7c12ac44283b516dac8cda12a5edeb</Sha>
+      <Sha>1b1a69573fab5799531bd60113592f293e6b9a66</Sha>
```

Comment:
For Fedora, we used commit `cd5a8c6` instead to pick up https://github.com/dotnet/source-build-reference-packages/pull/130